app/controllers/users/omniauth_controller.rb

```diff
@@ -13,5 +17,16 @@ def callback
         redirect_to users_none_url
       end
     end
+    # rubocop:enable Metrics/MethodLength
+
+    private
+
+    def allow_unregistered_government_user(received_email)
+      return if @user
+      allowed_tlds = Figaro.env.auto_account_creation_tlds.split(',')
+      return unless allowed_tlds.include?(received_email[-4..-1])
```

Comment:
personally I don't like numerically indexing a string like this.... what about a `.split('.').last` to get the TLD?

Reply:
And then maybe have the config be like `gov,mil` instead of `.gov,.mil` ?

Reply:
There was talk of allowing `.fed.us` so `.split('.').last` wouldn't work, but then, neither would `[-4..-1]`!  I'll revisit...

Reply:
@zachmargolis please take another look. This should work for all cases.